lib/node/client.mli

```diff
@@ -40,3 +36,5 @@ val request : node ref -> bytes -> Address.t -> Message.t Lwt.t
 of recipients and collects the responses in a list of [Message.t Lwt.t]. *)
 val broadcast_request :
```

Comment:
This one should be either renamed or deleted.
If we want to broadcast a message, we should use the dissemination. If we want to send direct message to everyone _(can't see any use case where such a feature would be useful)_, we should rename it.